src/routes/LobbyRouter.ts

```diff
@@ -29,12 +30,33 @@ enum ChannelType {
   Nojoin = 'private'
 }
 
-function generateLobbyName(type : ChannelType, owner : DiscordUser, guildUser : GuildUser) {
+function generateLobbyName(
+  type : ChannelType,
+  owner : DiscordUser,
+  guildUser : GuildUser,
+) : string {
   let icon : string;
   if (type === ChannelType.Nojoin) icon = '🔐';
   if (type === ChannelType.Mute) icon = '🙊';
   else icon = '🔊';
 
+  if (type === 'public') {
+    if (guildUser.tempName) {
+      const result = emojiRegex().exec(guildUser.tempName);
+      if (result && result[0] === guildUser.tempName.substr(0, result[0].length)) {
+        const [customIcon] = result;
+
+        if (customIcon !== '🔐' && customIcon !== '🙊') {
+          const name = guildUser.tempName
+            .substring(result[0].length, guildUser.tempName.length)
```

Comment:
Dit gaat er van uit dat de emoji altijd op positie 0 in de string zit

Reply:
Nee, wordt daarboven gecheckt of het wel de eerste karakters in de string zijn

Reply:
Ah, duidelijk!

Reply:
```ts
if (result && result[0] === guildUser.tempName.substr(0, result[0].length))
```